### PR TITLE
Reco add new commands

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Reco/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Reco
+
+- %%UPDATE_RN%%
```
Comment (daryakoval) [inline Packs/Reco/ReleaseNotes/1_0_2.md:6]: Please update release notes.

**Comment by daryakoval** [comment]
Thanks for the contribution!
Please add *reco-get-assets-user-has-access-to* command to the readme file. You can use - demisto-sdk generate-docs command. 

Please handle lint errors: https://app.circleci.com/pipelines/github/demisto/content/244364/workflows/212b3228-c8ec-4b37-858e-7ca3188acd45/jobs/690393

- /devwork/Reco.py:624:21: E0602 main: Undefined variable 'get_assets_by_owner'
- /home/circleci/project/Packs/Reco/Integrations/Reco/Reco.py:274:13: error:
Value of type "object" is not indexable  [index]
                params["getTableRequest"]["fieldFilters"]["fieldFilterGrou...
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~...
- /home/circleci/project/Packs/Reco/Integrations/Reco/Reco.py:624:22: error: Name
"get_assets_by_owner" is not defined  [name-defined]
                result = get_assets_by_owner(reco_client, demisto.args()["...
                         ^
Found 2 errors in 1 file (checked 1 source file)

Reco.py:287:29: E123 closing bracket does not match indentation of opening bracket's line
Reco.py:296:52: E226 missing whitespace around arithmetic operator
Reco.py:494:131: E501 line too long (144 > 130 characters)
Reco_test.py:400:23: E128 continuation line under-indented for visual indent
Reco_test.py:410:23: E128 continuation line under-indented for visual indent
Reco_test.py:413:64: W292 no newline at end of file

Thanks.

**Comment by edik24** [comment]
please fill the form:
https://github.com/demisto/content/pull/25322#issuecomment-1471461097

File: Packs/Reco/ReleaseNotes/1_0_2.md (hunk 1 above)
Comment (GalNakash-RecoLabs) [reply Packs/Reco/ReleaseNotes/1_0_2.md:6]: Fixed

**Comment by GalNakash-RecoLabs** [comment]
@daryakoval let me know if anything else is missing

**Review by edik24 — CHANGES_REQUESTED** [review]
Hi @GalNakash-RecoLabs please update the integration readme as well, and not only pack readme

**Comment by GalNakash-RecoLabs** [comment]
> 
Done

**Review by daryakoval — APPROVED** [review]

**Review by edik24 — APPROVED** [review]

**Pull request merged**